src/components/nodes/startEvent/startEvent.vue

```diff
@@ -48,13 +50,35 @@ export default {
     return {
       shape: null,
       definition: null,
+      showDropdown: false,
     };
   },
+  computed: {
+    isBaseStartEvent() {
+      return this.node.type === 'processmaker-modeler-start-event';
+    },
+  },
+  methods: {
+    removeDropdownOnUnhighlight() {
+      const unwatch = this.$watch('highlighted', highlighted => {
+        if (!highlighted) {
+          this.showDropdown = false;
+          unwatch();
+        }
+      });
+    },
+  },
   watch: {
     'node.definition.name'(name) {
       this.shape.attr('label/text', name);
     },
   },
+  created() {
+    if (this.isBaseStartEvent && this.highlighted) {
+      this.showDropdown = true;
+      this.removeDropdownOnUnhighlight();
```

Comment:
I'm not sure if having a separate method here really makes things much clearer. What doesn't anyone else think? Would this be clearer if it was inlined?